uftrace.c

```diff
@@ -180,6 +181,7 @@ __used static const char uftrace_help[] =
 "      --no-comment           Don't show comments of returned functions\n"
 "      --no-event             Disable (default) events\n"
 "      --no-sched             Disable schedule events\n"
+"      --no-preempt-sched     Hide pre-emptive schedule event\n"
```

Comment:
I'd like to have a common prefix like `--no-sched-preempt`.

Reply:
I change '--no-preempt-sched' to '--no-sched-preempt' following review.